payments/swish/direct.md

```diff
@@ -265,6 +264,88 @@ Content-Type: application/json
 
 {% include transaction-response.md showRequest=false payment_instrument="swish" transaction="sale" %}
 
+
+## Create M-Commerce Sale transaction
+
+This operation creates an m-commerce sales transaction in the direct payment
+scenario. This is managed either by sending a `POST` request as seen below, or
+by directing the end-user to the hosted payment pages. Note that the `msisdn`
+value (the end-user's mobile number) is left out in this request. The
+`redirect-app-swish` operation and the `paymentRequestToken` is only present in
+the m-commerce flow response, and the `view-qr-code` operation is only present
+in the m-commerce flow with QR code data specified.
+`paymentRestrictedToAgeLimit` must be set in create payment to be present.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/swish/payments/{{ page.payment_id }}/sales HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "transaction": {
+    }
+}
+```
+
+{:.code-header}
+**Response**
+```http
+GET /psp/{{ instrument }}/payments/{{ page.payment_id }}/{{ plural }} HTTP/1.1
+Host: {{ page.api_host }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+{
+    "payment": "/psp/swish/payments/{{ page.payment_id }}",
+    "sale": {
+        "date": "23.10.2017 08:39:37 +00:00",
+        "paymentRequestToken": "97aabad01f3a4b4282c4e4fa36c8c259",
```

Comment:
I don’t think we should document this publicly.

Reply:
I agree on this.